### ISSUE TITLE
You can click more than one time to clock SweetAlert

### ISSUE THREAD

**Issue by sibelius** [t4t5/sweetalert#454]
```
window.swal({
                      title: "",
                      text: "Do you want to do a Post?",
                      type: "warning",
                      showCancelButton: true,
                      confirmButtonText: "Yes",
                      cancelButtonText: "No",
                      closeOnConfirm: false,
                      closeOnCancel: false },
function(isConfirm) {
    console.log(isConfirm)
}
```

In this example, you can click more than on time on Yes or No (if you are fast enough), and then you will receive multiple logs or even worst, you can send multiple POST to your server

**Comment by Hybridx24** [comment]
I've found that using `showLoaderOnConfirm=True` can prevent this.

**Comment by t4t5** [comment]
Fixed in 2.0
